python-builder/bin/build

```diff
@@ -5,6 +5,6 @@ set -euo pipefail
 pipenv run python setup.py sdist > /dev/null 2>&1
 
 # Get package name so we can echo out the built artifact name
-PACKAGE_BUILD_NAME="$(ls dist/*.tar.gz)"
+PACKAGE_BUILD_NAME="$(pipenv run python setup.py --fullname).tar.gz"
```

Comment:
@KeltonKarboviak Can you explain a bit about `setup.py`, and why it needs run twice?

Reply:
The first run is what actually builds the package and creates a source tar (i.e. a .tar.gz file in the 'dist/' folder).

The second run simply outputs the name of the just built package with its version number (i.e., my-package-0.1.0).  Then I just manually append .tar.gz to get the full filename of the built source archive.